enterprise/frontend/src/metabase-enterprise/moderation/index.js

```diff
@@ -128,3 +134,14 @@ export function getModerationEvents(question, usersById) {
 export function isUserModerator(user) {
   return user.id === 1;
 }
+
+export function buildModerationRequestPath(request) {
+  const { moderated_item_type, moderated_item_id, id } = request;
+
+  switch (moderated_item_type) {
+    case "card":
+      return `/question/${moderated_item_id}?moderationRequest=${id}`;
```

Comment:
Should we use an URL with a slug here?

Reply:
Yes, thanks for reminding me -- fixed it here: https://github.com/metabase/metabase/pull/16285/commits/4559812bb8449fabe9397eb9088938b00a02d2b2

Reply:
Looks great, thanks for fixing it